requirements.txt

```diff
@@ -9,7 +9,7 @@ psutil >= 5.0
 sortedcontainers !=2.0.0, !=2.0.1
 tblib >= 1.6.0
 toolz >= 0.8.2
-tornado >= 6.0.3, <6.2
+tornado >= 6.2
```

Comment:
tiny drive-by comment, is the requirement to upgrade to `>=6.2` necessary here? I ask because I saw that in https://github.com/dask/distributed/pull/7165 there was code backported for when tornado < 6.2.0

Reply:
It will not be, but I want to ensure that this tests against 6.2 or later in CI for now.

Reply:
requirements.txt changes will translate directly to the packaged version. If you only want to force CI to use this, please only pin in `continuous_interation/environment*.yml`